packages/p2p/crowdin.yml

```diff
@@ -1,5 +1,5 @@
 project_id: '381909'
-api_token_env: 'DERIV_COM_CROWDIN_API_KEY'
+api_token_env: 'CROWDIN_API_KEY'
```

Comment:
Since some of us also handle translations for `binary-static`, it's going to use the same `CROWDIN_API_KEY` variable since we set the same variable name in our local `env` for both projects. Will this work fine?
Will the `CROWDIN_API_KEY` work with different `project_id`s?